packages/trpc/server/routers/viewer/workflows/getByViewer.handler.ts

```diff
@@ -94,20 +99,45 @@ export const getByViewerHandler = async ({ ctx }: GetByViewerOptions) => {
     },
   });
 
+  const teamMemberships = user.teams.map((membership) => ({
+    teamId: membership.team.id,
+    membershipRole: membership.role,
+  }));
+
   workflowGroups = ([] as WorkflowGroup[]).concat(
     workflowGroups,
-    user.teams.map((membership) => ({
-      teamId: membership.team.id,
-      profile: {
-        name: membership.team.name,
-        slug: "team/" + membership.team.slug,
-        image: `${CAL_URL}/team/${membership.team.slug}/avatar.png`,
-      },
-      metadata: {
-        readOnly: membership.role === MembershipRole.MEMBER,
-      },
-      workflows: membership.team.workflows,
-    }))
+    user.teams
+      .filter((mmship) => {
+        const metadata = teamMetadataSchema.parse(mmship.team.metadata);
+        return !metadata?.isOrganization;
```

Comment:
filter out orgs (workflows)

Reply:
If it's the same logic between workflows & webhooks, can we abstract this into it's own function?